doc/content/getting-started/users-orgs/_index.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
Maybe better to rename the folder to `users-organizations`

Reply:
kept it as `user-management` to save links